pkg/volume/aws_ebs/aws_util.go

```diff
@@ -116,6 +116,11 @@ func verifyDevicePath(devicePaths []string) (string, error) {
 
 // Unmount the global mount path, which should be the only one, and delete it.
 func unmountPDAndRemoveGlobalPath(globalMountPath string, mounter mount.Interface) error {
+	if pathExists, pathErr := pathExists(globalMountPath); pathErr != nil {
+		return fmt.Errorf("Error checking if path exists: %v", pathErr)
+	} else if !pathExists {
```

Comment:
> Is this expected that we are trying to unmount a path and it does not exist? Are we failing silently?

It is possible. Assume globalMountPath unmount succeeds but the OpenFile check afterwards fails (because there is a stray process holding on to a device mount in another namespace, for example) then subsequent calls to unmount will find no globalMountPath (since it has already been unmounted and removed).

> Are we failing silently?

Yes. We should `glog.V(5)` a warning message indicating unmounting is being skipped because the globalMountPath does not exist.

> Should this preamble be a util function as we are writing it three times?

Always good to have repeated code in one place instead of many.